torch_glow/src/PyTorchCommon.cpp

```diff
@@ -196,14 +197,53 @@ glow::Type ptTypeToGlowType(const c10::TensorType &ptType, float scale,
   return glow::Type(scalarTypeToElemKind(scalarType), dims, scale, zero_point);
 }
 
+at::Tensor requantizeToDtype(at::Tensor ptTensor, c10::ScalarType dtype) {
+  CHECK(ptTensor.is_contiguous());
+  if (dtype != at::kQInt8 && dtype != at::kQUInt8) {
+    LOG(DFATAL) << "Can only requantize to int8 or uint8";
+  }
+  if (!ptTensor.is_quantized()) {
+    LOG(DFATAL) << "Only support perform requantize in quantized tensor.";
+  }
+  if (ptTensor.qscheme() != at::kPerTensorAffine) {
```

Comment:
Do we have use case of channelwise quantized tensor? I think we do? And what's the difference in algorithm if the tensor is channelwise, same?

Reply:
No, I dont think we have use case of channelwise quantized tensor, since that only exist in packed_conv's weight, which is a packed tensor and we dont cut there in neither debugging or excluding unsupported ops(adaptive_avg_pool).

but you are right, the algorithm should be the same.